src/Microsoft.ML.Data/Data/DataViewUtils.cs

```diff
@@ -194,6 +193,29 @@ public static bool AllCachable(Schema schema, Func<int, bool> predicate)
             return true;
         }
 
+        /// <summary>
+        /// Return whether all the active columns, as determined by the predicate, are
+        /// cachable - either primitive types or vector types.
+        /// </summary>
+        public static bool AllCachable(Schema schema, IEnumerable<Schema.Column> colsNeeded)
```

Comment:
>Cachable [](start = 30, length = 8)

Please ignore my comment, unless you have some free time:
[https://lists.w3.org/Archives/Public/ietf-http-wg-old/1997SepDec/0003.html](https://lists.w3.org/Archives/Public/ietf-http-wg-old/1997SepDec/0003.html)

I also look in google index and "Cachable" appears 148 000 times while "Cacheable" 876 000.

Reply:
since you spent time looking this up, I'll rename it!

---
In reply to: [246952100](https://github.com/dotnet/machinelearning/pull/1975#discussion_r246952100) [](ancestors = 246952100)

Reply:
Please, don't this is not only place in code where we call it cachable, and i don't want you to spend your time on such minutiae.

---
In reply to: [247017070](https://github.com/dotnet/machinelearning/pull/1975#discussion_r247017070) [](ancestors = 247017070,246952100)

Reply:
Only reason why I left comment is link with description of rule and exception out of it for using or not using "e".

---
In reply to: [247017629](https://github.com/dotnet/machinelearning/pull/1975#discussion_r247017629) [](ancestors = 247017629,247017070,246952100)

Reply:
Rename does it for us, it is just 2 clicks and typing one letter !

---
In reply to: [247017629](https://github.com/dotnet/machinelearning/pull/1975#discussion_r247017629) [](ancestors = 247017629,247017070,246952100)